README.md

```diff
@@ -133,9 +133,8 @@ $parsed = $architect->parseData($books, [
 nice convenience methods to define the Architect relationships in query parameters.
 
 ## Installation
-
 ```bash
-composer require optimus/architect ~1.0
+composer require optimus/architect dev-laravel-6
```

Comment:
This is the only real change, I guess if you accept this PR, you'll name a new version didn't dare to decide that is V2.0 or so